### PR TITLE
scroll up

### LINKED ISSUE #3535: If an error occurs when sending a payment, scroll up to the top so that the error is actually visible (which is not always the case, e.g. if the advanced options area is opened)

### PR DESCRIPTION
**Pull request by tanojaja**
Fixes #3535 #3816

### CODE REVIEW

**Comment by gabrielbazan7** [comment]
I think #3816 is not solved.

**Comment by gabrielbazan7** [comment]
ACK #3535

**Comment by cmgustavo** [comment]
ACK, it works on iOS. 

Agree with @gabrielbazan7 , this PR doesn't solve #3826 .

**Comment by tanojaja** [comment]
Updated.

**Comment by cmgustavo** [comment]
I ‘m reviewing a little more, and I figured it out that the `master` branch works as expected without this PR. Please, could you review that and confirm if this PR is necessary? The issue is old, and I can check that after pressing the send button, the page is scrolled up automatically

**Comment by gabrielbazan7** [comment]
Discussing, we decided that is not necessary the scroll up in the send tab, considering that all the options fit correctly on the screen in a standard device and also the page is scrolled up automatically on ios.

**Comment by gabrielbazan7** [comment]
ACK

**Comment by tanojaja** [comment]
Updated

**Comment by gabrielbazan7** [comment]
ACK

File: public/views/includes/menu.html [hunk 1]
```diff
@@ -9,7 +9,7 @@
     <menu-toggle ng-show="index.menu.length > 6"></menu-toggle>
 </div>
 
-<div class="bottom-bar row collapse" ng-show="!index.notAuthorized">
+<div ng-hide="hideMenuBar" class="bottom-bar row collapse" ng-show="!index.notAuthorized">
```
Comment (cmgustavo) [inline public/views/includes/menu.html]: `hideMenuBar` should be in `walletHome.html` where this file is included.

**Comment by tanojaja** [comment]
Updated

**Comment by tanojaja** [comment]
Updated

**Comment by tanojaja** [comment]
Updated

**Comment by matiu** [comment]
Code ACK.

**Pull request merged**